src/tests/cpp/security/XmlAuthzCredentialGenerator.hpp

```diff
@@ -27,7 +27,7 @@
 
 #include <algorithm>
 #include <time.h>
```

Comment:
I also didn't `sed` this one since it wasn't in our publics.  I'll do that. Thanks